specification/dimensions/billingaccountid.md

```diff
@@ -0,0 +1,34 @@
+# Billing Account ID
+
+A billing account is a container for resources and/or services that are billed together in an invoice. Common scenarios for using billing accounts include grouping based on organizational constructs and hierarchy-based cost allocation approach.
+
+A billing account id is an provider assigned identifier for a billing account. It is commonly used for cost reporting, invoice reconciliation and cost allocation scenarios.
+
+The BillingAccountId column MUST be present in the billing data. This column must be of type String and MUST NOT contain null values. BillingAccountId SHOULD be a globally unique identifier within a provider. BillingAccountId MUST be unique when a customer has more than one billing account.
+
+See [Appendix: Grouping of resources and/or services](https://docs.google.com/document/u/0/d/1U24it_zByg-jKB9iOmiFbrULtzj5G5wUtawvntyeJuA/edit) section for billing accounts, sub-/resource accounts and hierarchy-based cost allocation approach scenarios.
```

Comment:
Don't link to Google Docs.

Reply:
I am removing the content around Appendix since its not added yet. I have added a comment in https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/issues/91 to add the link once its ready.